tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -71,6 +72,65 @@ def test_fsx_lustre(
     _test_data_repository_task(remote_command_executor, mount_dir, bucket_name, fsx_fs_id, region)
 
 
+@pytest.mark.regions(["us-east-1"])
+@pytest.mark.instances(["c5.xlarge", "m6g.xlarge"])
+@pytest.mark.schedulers(["sge"])
+@pytest.mark.usefixtures("os", "instance", "scheduler")
+# FSx is not supported on CentOS 6
+@pytest.mark.skip_oss(["centos6"])
+# FSx is only supported on ARM instances for Ubuntu 18.04 and Amazon Linux 2
+@pytest.mark.skip_dimensions("*", "m6g.xlarge", "alinux", "*")
+@pytest.mark.skip_dimensions("*", "m6g.xlarge", "centos7", "*")
+@pytest.mark.skip_dimensions("*", "m6g.xlarge", "ubuntu1604", "*")
+def test_fsx_lustre_backup(
```

Comment:
Nothing to comment here. I just think this is a very well done test, well engineered and well explained, and I wanted to let you know :)

Reply:
The credit for this belongs to @siddharthsalot, as does the credit for the rest of these changes.